content/articles/letsencrypt.md

```diff
@@ -58,7 +58,7 @@ Some Let's Encrypt features may not be supported by DNSimple. Check the [limitat
 The DNSimple Let's Encrypt integration allows you to request an SSL certificate for free using the Let's Encrypt certification authority.
 
 <note>
-To request an SSL certificate with Let's Encrypt, the domains **must be delegated to and resolving only with DNSimple**. The domain doesn't need to be registered with DNSimple. This means it is not compatible with Secondary DNS.
+To request an SSL certificate with Let's Encrypt, the domains **must be delegated to and exclusively resolving with DNSimple**, it is not compatible with Secondary DNS. The domain doesn't need to be registered with DNSimple.
```

Comment:
This is a comma splice now, you need a conjunction or a period. 
 
```suggestion
To request an SSL certificate with Let's Encrypt, the domains **must be delegated to and exclusively resolving with DNSimple**. It is not compatible with Secondary DNS. The domain doesn't need to be registered with DNSimple.
```

Reply:
thank you so much